data/mods/Tanks/items.json

```diff
@@ -0,0 +1,115 @@
+[
+    {
+        "type":"GENERIC",
+        "id" : "motor_electric_super",
+        "name" : "supercharged electric motor",
+        "description" : "A massive, high-efficiency electronic motor designed for atomic-engine military vehicles.",
+        "weight" : 73708,
+        "to_hit" : 0,
+        "color" : "light_cyan",
+        "symbol" : ",",
+        "material" : ["superalloy", "null"],
+        "volume" : 15,
```

Comment:
"massive"?
Also, superalloy is light rather than tough. Hardsteel would fit better.